tests/client-dbus/tests/misc/test_predict.py

```diff
@@ -44,11 +44,11 @@ def _call_predict_usage_pool(
     """
     with subprocess.Popen(
         [_STRATIS_PREDICT_USAGE, "pool"]
-        + ["--device-size=%s" % size.magnitude for size in device_sizes]
+        + [f"--device-size={size.magnitude for size in device_sizes}"]
```

Comment:
This one should be ```[f"--device-size={size.magnitude}" for size in device_sizes]```

Reply:
OK,  I added this and another fix as standalone commits; I can fold them into the formatted string commit.

Reply:
Are you sure about that?  If I try it, there's an `Undefined variable 'size'` error on linting.

Reply:
...wait, I found it: the `"` character is misplaced.